### ISSUE TITLE
3DPW eval  error

### ISSUE THREAD

**Issue by lml125** [naver/multi-hmr#52]
Hello, after reading your paper, I felt that I learned a lot. However, when I ran your evaluation code on the 3DPW test set, why is the result so different from yours? Here is the result display information:
/home/fang3090ti02/new/anaconda3/envs/multihmr/bin/python /home/fang3090ti02/new/multi-hmr-master (1)/multi-hmr-master/train.py --eval_only 1 --backbone dinov2_vitl14 --img_size 896 --val_data THREEDPW --val_split test --val_subsample 25 --pretrained models/multiHMR/multiHMR_896_L 
Using cache found in /home/fang3090ti02/.cache/torch/hub/facebookresearch_dinov2_main
/home/fang3090ti02/.cache/torch/hub/facebookresearch_dinov2_main/dinov2/layers/swiglu_ffn.py:43: UserWarning: xFormers is available (SwiGLU)
  warnings.warn("xFormers is available (SwiGLU)")
/home/fang3090ti02/.cache/torch/hub/facebookresearch_dinov2_main/dinov2/layers/attention.py:27: UserWarning: xFormers is available (Attention)
  warnings.warn("xFormers is available (Attention)")
/home/fang3090ti02/.cache/torch/hub/facebookresearch_dinov2_main/dinov2/layers/block.py:33: UserWarning: xFormers is available (Block)
  warnings.warn("xFormers is available (Block)")
  0%|          | 0/982 [00:00<?, ?it/s]
ARGS: Namespace(train_data='BEDLAM', train_split='training', train_n=-1, val_data=['THREEDPW'], val_split=['test'], val_n=[-1, -1, -1], val_subsample=[25], save_dir='logs', name='trainval', pretrained='models/multiHMR/multiHMR_896_L', batch_size=2, train_subsample=1, num_workers=0, img_size=896, backbone='dinov2_vitl14', n_iters_per_epoch=100, log_freq=100, max_iter=10000, nb_max_ckpt=10, amp=1, use_efficient_attention=1, learning_rate=5e-06, eval_only=1, person_center='head', visu_to_save=0, extension='png', res=None, num_betas=10, det_thresh=0.2, nms_kernel_size=3, crops=[0], flip=1, brightness=0.0, contrast=0.0, saturation=0.0, hue=0.0, alpha_bce=10.0, alpha_offset=1.0, alpha_rotmat=0.1, alpha_shape=1.0, alpha_dist=1.0, alpha_transl=1.0, alpha_j3d=100.0, alpha_v3d=100.0, alpha_j2d=1.0, alpha_v2d=1.0, start_2d_epoch=10, max_epochs=100, log_dir='logs/trainval', ckpt_dir='logs/trainval/checkpoints', visu_dir='logs/trainval/visu')
LOG_DIR: logs/trainval

EVAL: 
  0%|          | 1/982 [00:00<12:21,  1.32it/s]i=0 - Recall=100.0 - PVE=633.2 - PA-PVE=405.1 - MPJPE=603.2 - PA-MPJPE=403.8
  4%|▎         | 36/982 [00:07<02:37,  5.99it/s]/home/fang3090ti02/new/multi-hmr-master (1)/multi-hmr-master/utils/training.py:186: RuntimeWarning: overflow encountered in half_scalars
  bb1_area = (bb1['x2'] - bb1['x1'] + 1) * (bb1['y2'] - bb1['y1'] + 1)
 10%|█         | 101/982 [00:18<02:27,  5.97it/s]i=100 - Recall=100.0 - PVE=625.4 - PA-PVE=405.1 - MPJPE=579.6 - PA-MPJPE=406.9
 20%|██        | 201/982 [00:34<02:09,  6.03it/s]i=200 - Recall=100.0 - PVE=631.6 - PA-PVE=399.7 - MPJPE=589.0 - PA-MPJPE=402.3
 31%|███       | 301/982 [00:50<01:47,  6.33it/s]i=300 - Recall=100.0 - PVE=628.0 - PA-PVE=397.2 - MPJPE=586.8 - PA-MPJPE=399.5
 41%|████      | 401/982 [01:07<01:33,  6.22it/s]i=400 - Recall=100.0 - PVE=627.2 - PA-PVE=395.1 - MPJPE=587.4 - PA-MPJPE=396.7
 51%|█████     | 501/982 [01:24<01:16,  6.33it/s]i=500 - Recall=99.0 - PVE=629.4 - PA-PVE=389.2 - MPJPE=588.0 - PA-MPJPE=389.8
 61%|██████    | 601/982 [01:40<00:59,  6.42it/s]i=600 - Recall=99.0 - PVE=627.6 - PA-PVE=388.6 - MPJPE=587.3 - PA-MPJPE=388.6
 71%|███████▏  | 701/982 [01:56<00:50,  5.58it/s]i=700 - Recall=100.0 - PVE=627.0 - PA-PVE=388.9 - MPJPE=587.1 - PA-MPJPE=389.9
 82%|████████▏ | 801/982 [02:14<00:31,  5.68it/s]i=800 - Recall=100.0 - PVE=628.8 - PA-PVE=388.0 - MPJPE=589.6 - PA-MPJPE=388.1
 92%|█████████▏| 901/982 [02:29<00:12,  6.47it/s]i=900 - Recall=100.0 - PVE=626.1 - PA-PVE=385.7 - MPJPE=587.5 - PA-MPJPE=385.8
100%|██████████| 982/982 [02:42<00:00,  6.03it/s]
***EVAL METRICS - 3dpw-test-25***
    - pve: 625.1
    - pa_pve: 386.1
    - precision: 0.0
    - recall: 100.0
    - f1_score: 1.0
    - mpjpe: 587.0
    - pa_mpjpe: 386.1

Process finished with exit code 0

**Comment by fabienbaradel** [comment]
Hi, thank you for your interest in ou work.
There is an error while loading the pretrained model. You are not giving the right path. It should be ```--pretrained models/multiHMR/multiHMR_896_L.pt``` (do not forget the . pt at the end).
Best,
